CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
+## TBD
+
+### Changed
+
 ## 7.10.3 (2021-06-15)
 
+- (react-native): Update bugsnag-cocoa to v6.9.5
```

Comment:
think this has gone under the wrong heading

Reply:
🤦